modules/installation-gcp-user-infra-adding-ingress.adoc

```diff
@@ -16,11 +16,15 @@ ifdef::shared-vpc[]
 = Adding the ingress DNS records
 endif::shared-vpc[]
 
-If you removed the DNS Zone configuration when creating Kubernetes manifests and
-generating Ignition configs, you must manually create DNS records that point at
-the ingress load balancer. You can create either a wildcard
-`*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other
-records per your requirements.
+DNS Zone configuration is removed when Kubernetes manifests and generates Ignition configs. You must manually create DNS records that point at the ingress load balancer. You can create either a wildcard
+`*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other records per your requirements.
+
+ifdef::shared-vpc[]
+[NOTE]
+====
+This sections covers an optional step in other types of installation but it is mandatory for shared vpn install. Therefore, this change should NOT be propagated to other installation types.
```

Comment:
Singular: s/sections/section/

s/installation but/installations, but/

Reply:
Is "vpn" in the BZ maybe a typo of "VPC"? So possibly:

s/vpn install/VPC installations/

Reply:
Re: "NOT", IBM SG states to avoid using uppercase for emphasis and allows italics. Our [repo guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) also allow italics, though suggests using sparingly.

Reply:
Re-reading the BZ now, actually, I'm wondering if this note was intended to be included verbatim in the doc, or was just describing the suggested paragraph change?

Reply:
Yes, 'vpn' should be 'vpc' - sorry. I edited the BZ.

And yes, that sentence beginning with 'Note:' is just a reminder that changes for the BZ are not supposed to go to other pages like Restricted Network or regular UPI install pages.

Hope this helps. Thanks.

Reply:
I removed this note from the documentation. I thought it was meant for the doc and not for us :-)